docs/en/setup/service-agent/java-agent/Application-toolkit-micrometer.md

```diff
@@ -0,0 +1,44 @@
+* Dependency the toolkit, such as using maven or gradle
+```xml
+   <dependency>
+      <groupId>org.apache.skywalking</groupId>
+      <artifactId>apm-toolkit-micrometer-registry</artifactId>
+      <version>${skywalking.version}</version>
+   </dependency>
+```
+
+* Using `SkywalkingMeterRegistry` as the registry, it could automatic collect your meter to out meter system.
```

Comment:
`SkywalkingMeterRegistry` should be a full name, including package too.

```suggestion
* Using `SkywalkingMeterRegistry` as the registry, it could forward the MicroMeter collected metrics to OAP server.
```